### PR TITLE
Working on #2972 Template Conversion to Twig Format (member.php)

### PR DESCRIPTION
**Pull request by PaulBender**
#2972 for member.php

This includes all member.php pages except for the registration and profile pages (due to how complex they are; I'll do those as separate PRs).

### CODE REVIEW

File: inc/views/base/member/activate.twig [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{% endblock head %}
+
+{% block body %}
+    <form action="member.php" method="post">
+        <table border="0" cellspacing="{{ theme.borderwidth }}" cellpadding="{{ theme.tablespace }}" class="tborder">
+            <tr>
+                <td class="thead" colspan="2"><strong>{{ lang.activate_account }}</strong></td>
+            </tr>
+            <tr>
+                <td class="trow1" width="30%"><strong>{{ lang.username }}</strong></td>
```
Comment (euantorano) [inline inc/views/base/member/activate.twig]: Can you wrap this in a `<label for="username">` and add `id="username"` to the associated input? I've noticed a lot of inputs don't have labels linked and it's useful for accessibility to add them. If they're already there before we convert to the new theme, it makes it harder to miss them 😄

File: inc/views/base/member/emailuser.twig [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            {% endif %}
+            <tr>
+                <td width="40%" class="trow2"><strong>{{ lang.email_subject }}</strong></td>
+                <td width="60%" class="trow2"><input type="text" class="textbox" size="50" name="subject" value="{{ email.subject }}" /></td>
+            </tr>
+            <tr>
+                <td valign="top" width="40%" class="trow1"><strong>{{ lang.email_message }}</strong></td>
+                <td width="60%" class="trow1"><textarea cols="50" rows="10" name="message">{{ email.message }}</textarea></td>
+            </tr>
+            {{ captcha|raw }}
```
Comment (euantorano) [inline inc/views/base/member/emailuser.twig]: Captcha is fairly common, perhaps we should make a Twig function for it - thoughts?

**Comment by euantorano** [comment]
Looks good, just a very minor change 👍

**Comment by PaulBender** [comment]
I've added label and ids to all input fields.

File: inc/views/base/member/emailuser.twig (hunk 2 above)
Comment (PaulBender) [reply inc/views/base/member/emailuser.twig]: Definitely should be one.

**Comment by euantorano** [comment]
Brilliant, thanks!

**Pull request merged**